src/dary/variant.rs

```diff
@@ -8,6 +8,12 @@ pub struct Dary<const D: usize, T> {
     p: PhantomData<T>,
 }
 
+/// Tree variants do not hold any data; and hence, safe to sync.
+unsafe impl<const D: usize, T> Sync for Dary<D, T> {}
+
+/// Tree variants do not hold any data; and hence, safe to sync.
+unsafe impl<const D: usize, T> Send for Dary<D, T> {}
```

Comment:
Might wanna make the documentation consistently either include or exclude having a `# Safety` header